### PR TITLE
feat: swiftlint 0.50.0 update

### PR DESCRIPTION
**Pull request by apps4everyone**
swiftlint 0.50.0 update

### CODE REVIEW

File: lib/version.rb
Comment (mvandervelden) [inline lib/version.rb:6]: The hash should also be updated, see issue #178  and PR #179.

**Review by mvandervelden — CHANGES_REQUESTED** [review]
Thanks for this PR!
From my last experience updating SwiftLint here, I learned the hard way to also update the `SWIFTLINT_HASH` var, for it to work properly.

File: lib/version.rb
Comment (apps4everyone) [reply lib/version.rb]: Hey @mvandervelden, thx for the quick feedback. Maybe you could provide me the steps to create the SWIFTLINT_HASH correctly, did: md5 portable_swiftlint.zip

**Comment by apps4everyone** [comment]
How would be now the release build and gem update to make the release visible? Our CI/CD is currently blocked because brew updated swiftlint to `0.50.0` but danger is using `0.46.2`. This miss match causes one of them failing.

**Review by mvandervelden — APPROVED** [review]
Thanks! _edit: I was a bit quick, missed your comments_ :)

File: lib/version.rb
Comment (mvandervelden) [reply lib/version.rb]: That should be correct yes. I just double checked on the version that I did.

**Comment by mvandervelden** [comment]
> How would be now the release build and gem update to make the release visible? Our CI/CD is currently blocked because brew updated swiftlint to 0.50.0 but danger is using 0.46.2. This miss match causes one of them failing.

If you want to unblock before this PR is merged & released, you can probably point your Gemfile to your forked repo & commit.

**Review by mvandervelden — APPROVED** [review]

**Comment by apps4everyone** [comment]
Should I merge now or is this done by the @ashfurrow?

**Review by ashfurrow — APPROVED** [review]
Let’s do it! I’ll try to get this released by tomorrow.

**Comment by apps4everyone** [comment]
Thank you so much, great work ❤️

**Comment by ashfurrow** [comment]
This has been released as `0.31.0`. Thanks again!

**Comment by apps4everyone** [comment]
Thx

**Comment by hlung** [comment]
How do I make it use the globally installed `swiftlint`, and not the version specified here?

**Pull request merged**
